modules/contacts/client/components/TrContactsCommon.component.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+// import { NamespacesConsumer } from 'react-i18next';
+import PropTypes from 'prop-types';
+import Contact from './Contact.component';
+// import { contactsCommonListService } from '@/modules/contacts/client/service/contacts-common-list.client.service';
+import { contactsCommonListService } from '../services/contacts-common-list.client.service';
+import '@/config/lib/i18n';
+
+export default class TrContactsCommon extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      contacts: []
+    };
+    this.isContactsCommonList = this.isContactsCommonList.bind(this);
+    this.onChange = this.onChange.bind(this);
+  }
+  isContactsCommonList() {
+    if (this.state.contacts.length === 0) {
+      return;
+    }
+  }
+  async onChange() {
+    const contacts = this.props.contacts;
+    const list = await contactsCommonListService(contacts);
+    console.log(list[0]);
+    this.setState({
+      contacts: list
+    });
+  }
+
+  componentDidMount() {
+    this.onChange();
+  }
+  render() {
+    this.isContactsCommonList();
+    return (
+      <div className="panel panel-default">
+        {/* convert ng-pluralize with NamespacesConsumer */}
+        <div className="panel-body">
+          <div className="contacts-contact">
+            {this.state.contacts.map((contact) => {
+              console.log(contact.user.displayName);
+              return (
+                <div key={contact}>
```

Comment:
Perhaps `contact._id` could be better suited as a key here. I'm guessing the contact here is an object? We may rather need to [provide a string](https://reactjs.org/docs/lists-and-keys.html#keys). Not sure if it matters though.

Reply:
@mrkvon 
I changed key={contact} to key={contact._id.toString()}